.travis.yml

```diff
@@ -3,5 +3,20 @@ node_js:
   - "6"
   - "8"
   - "10"
+services:
+  - mysql
+env:
+  global:
+    - TEST_MYSQL_HOST=localhost
+    - TEST_MYSQL_PORT=3306
+    - TEST_MYSQL_USER=test
+    - TEST_MYSQL_PASSWORD=test
+    - MYSQL_HOST=localhost
+    - MYSQL_PORT=3306
+    - MYSQL_USER=test
+    - MYSQL_PASSWORD=test
 before_script:
-  - "mysql -e 'create database myapp_test;'"
+  - mysql -u root -e "CREATE USER 'test'@'localhost' IDENTIFIED BY 'test';"
+  - mysql -e "use mysql; update user set authentication_string=PASSWORD('test') where User='test'; update user set plugin='mysql_native_password';FLUSH PRIVILEGES;"
```

Comment:
I wonder where the database `mysql` is created.  
From this line https://github.com/strongloop/loopback-connector-mysql/blob/master/test/init.js#L17, it seems like we are using `myapp_test` by default?

Reply:
looking at https://docs.travis-ci.com/user/database-setup/#mysql, looks like `mysql` is a default schema for user/system management.

`myapp_test` is the schema created for tests.
cc @dhmlau just my understanding :-p

Reply:
thx